### ISSUE TITLE
CMake Deprecation Warning

### ISSUE THREAD

**Issue by mxa** [libpd/libpd#321]
using CMake to compile libpd we get the following warning:

```
CMake Deprecation Warning at /libpd/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.

```

To get rid of this warning we could change the line 1 of `CMakeLists.txt` from 

`cmake_minimum_required (VERSION 2.8.11)`

to 

`cmake_minimum_required (VERSION 2.8.12)`

**Comment by mxa** [comment]
made a pull request https://github.com/libpd/libpd/pull/323
feel free to reject this, it's rather cosmetic. On the other hand, the fewer warnings the better :)

**Comment by danomatika** [comment]
Closed via #323
